src/routes/safe/components/Balances/SendModal/screens/AddressBookInput/index.jsx

```diff
@@ -0,0 +1,121 @@
+// @flow
+import React, { useState, useEffect } from 'react'
+import {
+  withStyles,
+} from '@material-ui/core/styles'
+import { useSelector } from 'react-redux'
+import Autocomplete from '@material-ui/lab/Autocomplete'
+import TextField from '@material-ui/core/TextField'
+import makeStyles from '@material-ui/core/styles/makeStyles'
+import { styles } from './style'
+import { getAddressBookListSelector } from '~/logic/addressBook/store/selectors'
+import { mustBeEthereumAddress, mustBeEthereumContractAddress } from '~/components/forms/validator'
+import Identicon from '~/components/Identicon'
+
+
+type Props = {
+  classes: Object,
+  fieldMutator: Function,
+  isCustomTx?: boolean,
+  recipientAddress?: string,
+}
+
+
+const textFieldLabelStyle = makeStyles(() => ({
+  root: {
+    overflow: 'hidden',
+    borderRadius: 4,
+    fontSize: '15px',
+    width: '500px',
+  },
+}))
+
+const textFieldInputStyle = makeStyles(() => ({
+  root: {
+    fontSize: '13.5px',
```

Comment:
why not 14?

Reply:
There is a problem with the text field component, if I use 14 as font size it won't wrap the text, dind't found a solution, do you have any ideas?. 
![image](https://user-images.githubusercontent.com/21086218/72800637-91289080-3c26-11ea-924a-be9f88ec469f.png)

Reply:
This wasn't the case with previous component. I think you may have to overwrite some material ui styles (maybe padding)

Reply:
I managed to make it work, with extend the width of the text field